moto/settings.py

```diff
@@ -32,6 +32,11 @@
 
 PRETTIFY_RESPONSES = bool(os.environ.get("MOTO_PRETTIFY_RESPONSES", False))
 
+# Fully skip test that require docker
+SKIP_REQUIRES_DOCKER = bool(os.environ.get("TESTS_SKIP_REQUIRES_DOCKER", False))
+# Run test that require docker, but allow failing when docker exceptions are raised
+RAISE_DOCKER_EXCEPTION = bool(os.environ.get("TESTS_XFAIL_REQUIRES_DOCKER", False))
```

Comment:
And this setting

Reply:
hmm, that should have been gone already. Will check.